lib/mobility/plugins/original_column.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Mobility
+  module Plugins
+    module OriginalColumn
+      extend Plugin
+
+      default false
+
+      requires :backend, include: :before
+    end
+
+    register_plugin(:original_column, OriginalColumn)
+  end
```

Comment:
I haven't really documented this anywhere yet, but this is a kind of convention/trick of defining a global plugin which does nothing, then defining a corresponding ORM-specific plugin which depends on the global one, and which is included automatically when you include the `active_record` plugin. So it's included but disabled by default.

That way, in your plugins, enabling the empty base plugin (`original_column`) indirectly enables the ORM-specific one. If in the future there was a corresponding plugin for Sequel, the same name could be used for that one (this is how `dirty`, `cache` and `query` work).